site/src/components/message.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Message
+description: The message component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fproto%2FamgDNGgGYy2v2Unc6Eet46%2FQ%2526A-Private-Messages%3Fpage-id%3D0%253A1%26type%3Ddesign%26node-id%3D337-533145%26viewport%3D-15969%252C-17033%252C0.75%26scaling%3Dmin-zoom%26starting-point-node-id%3D337%253A548136%26show-proto-sidebar%3D1%26mode%3Ddesign%26t%3DXzNRwbG95jzc2Fvd-1
+classType: Component
+classPrefixes:
+  - messages
+---
+
+# Message
+
+Use messages component for chat or messaging communications.
+
+## Usage
+
+Here is an example of a standard usage of messages component.
+
+```html
+<section class="display-grid gap-sm">
```

Comment:
Thinking we should structure the docs a little differently to match other components more closely:

1. Section for standard `message` component.
2. Section for displaying `message-right` (sidebar: is there a different name we could brainstorm here? not blocking)
3. Link off to `patterns/message-threads.md` a new page that lives in the patterns folder. On this page, can provide the recommended markup for a proper message thread, similar to what you have here.

Why? Component documentation should be focused and (generally) only contain markup related to that component. (For or no atomics)

The patterns folder is a place to show how to leverage any/all aspects of the framework to build a particular pattern. You can use various components together, apply atomics, etc.